workspace/TS100/Core/Src/GUIThread.cpp

```diff
@@ -352,27 +353,28 @@ static void gui_solderingTempAdjust() {
 	}
 }
 
-static int gui_SolderingSleepingMode() {
+static int gui_SolderingSleepingMode(uint8_t stayOff) {
```

Comment:
Why is this not a `bool` ?
its the same memory footprint, but `boot` is more clear as to its intent.